buyTogetherApp/templates/buyTogetherApp/main.html

```diff
@@ -0,0 +1,131 @@
+          </table>
+          <p class="bigFont">
+              We will all be working and helping each other in each field and each division but basically ,2 will be working on the front end and 2 on the back end and one on the communication between the group and QR checking codes and all together will be managing the database and models.
+          </p>
+          <p class="bigFont">
+           ***** Contributing will be more detailed on the Contributing.md *****<br>
+            <span class="red">Submitting a pull request:<br></span>
+                --(<span class="blue">Fork and clone the repository </span>:<br>
+                -Create a new branch. Please use the conventional commit formatting for the branch name.<br>
+                -git checkout -b feat-my-branch-name<br>
+
+                -Make your changes and preview the page using the local build.<br>
+                -Push to your fork and submit a pull request. Be sure to use the PR template.<br>
+                -Here are a few things you can do that will increase the likelihood of your pull request being accepted:<br>
+
+                -Review the documentation.<br>
+                -Follow the style/format of the existing code.<br>
+                -Keep your change as focused as possible; if there are multiple changes you would like to make that are not dependent upon each other, consider submitting them as separate pull requests.
+                -Be responsive during code review, as much as possible.)--<br>
+                --(<span class="blue">Commiting:</span><br>
+                            Welcome! We're so excited that you're considering contributing to the official beyond-group3 page for Red Hat Project.<br>
+                            Before submitting a pull request, please verify that:<br>
+                            -The commit message is in the format "- Description -".<br>
+                            -There are tests added to make sure the bug does not return/feature does not break in the future using flake8 one of the options.)--<br>
+                            --(<span class="red">Submitting an Issue:</span><br>
+                            <span class="blue">Bug Reporting:</span><br>
+                Here are a few helpful tips when reporting a bug:<br>
+
+                To report a bug, please use the bug report template.<br>
+                Make sure what you are describing is a bug and not a feature request. Is this how it was originally designed to work?<br>
+                Screenshots are very helpful in diagnosing bugs; please include them whenever possible.<br>
+                Check the console for errors and copy or snapshot the output. Please document this in your issue.)--<br>
+
+          </p>
+          <p class="bigFont ">
+              <span class="red">Meetings:We are planning to talk to each other all the time in the communication channels but face to face or online calls will be 3 times each week and each call for 2hrs+</span>
```

Comment:
That's a lot of long meetings :)
I hope they'll be productive :)

Reply:
We will be :)